hapi/src/api/account.api.js

```diff
@@ -135,7 +135,7 @@ const createLifebank = async ({
   }
 }
 
-const getProfile = async account => {
+const getProfile = async (account) => {
```

Comment:
This is coming form the CodeFactor, not sure about the reason, but it is what's happening